tools/nnyeah/nnyeah/nnyeah.csproj

```diff
@@ -26,4 +34,25 @@
       <DependentUpon>Errors.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
+  <Target Name="IncludeLegacy" BeforeTargets="_GetPackageFiles">
+    <PropertyGroup>
+      <ProjectRoot>$(MSBuildProjectDirectory)/../../..</ProjectRoot>
+      <LegacyiOSRoot>$(ProjectRoot)/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono</LegacyiOSRoot>
+      <LegacyMacRoot>$(ProjectRoot)/_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono</LegacyMacRoot>
+      <NetRoot>$(MSBuildProjectDirectory)/../../../_build</NetRoot>
+    </PropertyGroup>
+    <ItemGroup>
+      <Content Include="$(LegacyiOSRoot)/Xamarin.iOS/Xamarin.iOS.dll;$(LegacyMacRoot)/Xamarin.Mac/Xamarin.Mac.dll">
+        <Pack>true</Pack>
+        <PackagePath>content/legacy</PackagePath>
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      </Content>
+      <Content Include="$(NetRoot)/Microsoft.iOS.Ref/ref/net6.0/Microsoft.iOS.dll;$(NetRoot)/Microsoft.macOS.Ref/ref/net6.0/Microsoft.macOS.dll">
+        <Pack>true</Pack>
+        <PackagePath>content/net6</PackagePath>

```

Comment:
Could this path be changed to not hardcode "6" too?

Reply:
See ^ comment